### DISCUSSION TITLE
Fargate timed out error

### DISCUSSION THREAD

**Discussion by Mathdon** [artilleryio/artillery#2426] [Artillery]
Hi, I tired running a test in eu-west-w 2 on fargate and had this error

Error: Timed out waiting for workers to sync

It linked me to this bit of the code:

https://github.com/artilleryio/artillery/blob/main/packages/artillery/lib/platform/aws-ecs/legacy/run-cluster.js#L1742

Do you know what would cause this?

Thanks

**Comment by bernardobridge** [comment]
Hi @Mathdon 👋 ,

Can you provide a little bit more details ?
- How many workers were you running with ? (`--count`)
- What OS were you using?
- Did your script have any dependencies (e.g. custom javascript file, additional npm modules)?

We are currently aware of an intermittent issue happening with Fargate and are investigating, but without more details it's hard to know if it's connected.

Thank you!

**Reply by Mathdon** [reply]
HI @bernardobridge 

The script me run was

artillery run-fargate --subnet-ids subnet-1,subnet-2 --region eu-west-2 profiles/fargate.yml

I was on a mac OS Sonoma 14.2.1 

Yes I was using a custom.js script

The structure was

performance test project
    profiles
        fargate.yml
    plugins
        custom.js
        package.json
        package-lock.json
     data
       data.csv
package.json
package-lock.json

 In the custom.js file I was the npm packages axios and luxon which were included in the package.json file in the plugins directory
 
 Then just artillery was in the top level package.json

**Reply by Mathdon** [reply]
I was using version 2.0.4 of artillery to work with eu-west-2

**Reply by bernardobridge** [reply]
Thanks for the details! Just two more questions:

- were you encountering the error every single time you were running the command, or just sometimes?
- did you have this setup working with previous versions of Artillery? (i.e. did the problem start happening on 2.0.4?)

**Reply by Mathdon** [reply]
I have tried it twice and had the same error both times.

I have only just tried doing this, eu-west-2 was only supported in 2.0.4 so first time I am trying it

**Reply by bernardobridge** [reply]
I see @Mathdon . It's possible that this isn't the same error we're aware of, since that error seems to happen with higher `--count` and intermittently. Your case may just be a misconfiguration or issue with your particular test setup. 

I recommend checking the logs in AWS Cloudwatch for your test run. You can access them by:
1. Go to Cloudwatch in AWS, in the region you're using. Then click `Log groups`
2. Search for the Artillery log group (`artilleryio-log-group/artilleryio-cluster`) and click it.
3. You'll now be in the `Log Streams` tab, which will be ordered by Last event time. 
4. The streams you need will be in the format `artilleryio/<testID>/artillery/<taskID>`. If you search by the `testID` (it's outputted when you ran the test), it should show you 2 of them (assuming you ran 1 worker, which is the default if you didn't use `--count`). 
5. You need to select the one that has `artillery` in the name (not `datadog`).

Once inside the log stream, it should show you the logs from the worker. Please scan those for anything that looks like an error. Thank you!

**Reply by Mathdon** [reply]
Thanks @bernardobridge 

Do the logs get deleted as part of the clean up job that happens?

Couldn't see any logs in cloudwatch

**Reply by bernardobridge** [reply]
Hi @Mathdon .

The logs should remain there for a while. Are you sure you were looking in the same AWS region you ran your test in - `eu-west-2`?

**Reply by Mathdon** [reply]
Hi @bernardobridge 

Yeah I was in the correct region

No artillery logs

I attempted this yesterday so they shouldn't have been deleted yet

**Reply by samhunter1** [reply]
Hi,

Did you have any luck fixing this issue? I have a similar one in that my workers aren't syncing and also my log groups aren't creating any log streams. 

Thanks

**Comment by mrister** [comment]
I seem to have the same issue, it was working just a few days ago without any issue. I set --count 30, fargate allocates 28 or 29 and all the other workers are waiting (also incurring cost) for the last one(s) until it times out. 

This seems like a serious issue. Happens when I set count to 50 , 100 , does nto matter.

Also the --tags with run-fargate do not seem to get applied neither to the cluster nor taks..

**Reply by bernardobridge** [reply]
Thanks for the additional information, @mrister. Did this start happening when updating to the new Artillery version (2.0.4)? When you say it was working a few days ago, was that using an older Artillery version?

We're still working on pinpointing the cause for the issue.

---

> Also the --tags with run-fargate do not seem to get applied neither to the cluster nor taks..

Currently `tags` is a concept associated with the test run itself (and mostly for use with [Artillery Cloud](https://www.artillery.io/docs/cloud/get-started)). We may look to tag the created resources too in the future, though!

**Reply by hassy** [reply]
> I seem to have the same issue, it was working just a few days ago without any issue. I set --count 30, fargate allocates 28 or 29 and all the other workers are waiting (also incurring cost) for the last one(s) until it times out.

The regression is puzzling and I'm not sure what's caused it. It might have to do with a change in how `aws s3 sync` works in a recent release but I haven't confirmed it.

The waiting behavior is by design. All workers need to start executing the test at the same time so you've got to wait for everyone to get ready.

**Reply by mrister** [reply]
It was working with 2.04 and version before that, but now both of them fail if I set --count > 10.  For --count 10 it seems to work interestingly.

@bernardobridge YEah tags would be great as it would be easier to monitor the spend, do some aws reporting around them.
I understand that the tasks wait fo rall of them that are up and running, but in this case I get 49 running and everything stalls and waits and I could have used that time to do some testing even with 49 of em. So think about this asa configurable feature maybe, where you know you can say, ok I am fine with 90% of those..or similar logic.

**Comment by hassy** [comment]
Tracking as an issue in #2427

**Comment by hassy** [comment]
A fix is available in the canary release now:

Install the canary:

```sh
npm install -g artillery@2.0.4-a0b6328
```

And to run a test on Fargate, you'll need to tell Artillery to use the updated Docker image:

```sh
WORKER_IMAGE_URL=public.ecr.aws/d8a4z9o5/artillery-worker:a0b6328719a1223118c362ab923476b97f5a1e83 artillery run-fargate myscript.yml
```

All built from the most recent commit on `main` here: https://github.com/artilleryio/artillery/commit/a0b6328719a1223118c362ab923476b97f5a1e83

**Reply by mrister** [reply]
Oh, this seems to work!

**Reply by hassy** [reply]
@mrister that's great to hear! what's the largest test you've run so far?

**Reply by mrister** [reply]
@hassy about 400 containers, maxVusers = 20. So this did take a while for all of them to get up and finish running, that is kinda of something I would like to have avoided since my goal is to stress test a site with a maximum number of simultaneous users, and I think these were spread out over 30-40mins so I may not hit the max I was aiming for.

**Reply by hassy** [reply]
@mrister happy to take a look at your `config.phases`. The workers all start at the same time, so the load generated is not affected by startup time. If you're not getting the number of transactions you might need to tweak the phase definitions. Another thing to try if you're running Playwright tests especially is to bump CPU/memory to be able to run more VUs concurrently: https://docs.art/reference/cli/run-fargate#configuring-vcpu-and-memory-with---cpu-and---memory--

**Comment by En3Tho** [comment]
@hassy Hey! My issue isn't exactly the same as in the topic but still it is Fargate related.
What I've done:
1. Cloned repo
2. Went to examples/browser-load-testing-playwright
3. Did npm install
4. Tested local artillery using `npx artillery run browser-load-test.yml` with default settings (except for longer test run - 60 seconds)d
5. Did all the AWS IAM/Roles prep work for Fargate
6. Then went with `npx artillery run-fargate browser-load-test.yml`
7. Got this kind of log with "Timed out waiting for go signal" at the end (logs are from ecs so latest first). What did I do wrong here? Am I supposed to make a separate project with specific dependencies?

```
January 25, 2024 at 17:19 (UTC+3:00)	............................................................................................................................................................................................................................................................................................................Timed out waiting for go signal	-
January 25, 2024 at 17:07 (UTC+3:00)	Worker 1ca155c9bdaa4e40ad95f0b6b530cb62 synced up & ready	-
January 25, 2024 at 17:07 (UTC+3:00)	******** [1ca155c9bdaa4e40ad95f0b6b530cb62] Waiting for green signal	-
January 25, 2024 at 17:07 (UTC+3:00)	Waiting... (s3://artilleryio-test-data-431817564189/test-runs/tzhty_t8yeqcda9x5dzkay6j5bczzwgxffz_5hwy/go.json)	-
January 25, 2024 at 17:07 (UTC+3:00)	.	-
January 25, 2024 at 17:07 (UTC+3:00)	|-- browser-load-test.yml	-
January 25, 2024 at 17:07 (UTC+3:00)	|-- flows.js	-
January 25, 2024 at 17:07 (UTC+3:00)	|-- metadata.json	-
January 25, 2024 at 17:07 (UTC+3:00)	`-- package.json	-
January 25, 2024 at 17:07 (UTC+3:00)	0 directories, 4 files	-
January 25, 2024 at 17:07 (UTC+3:00)	******** [1ca155c9bdaa4e40ad95f0b6b530cb62] Ready to run	-
January 25, 2024 at 17:07 (UTC+3:00)	Completed 366 Bytes/366 Bytes (2.8 KiB/s) with 1 file(s) remaining move: s3://artilleryio-test-data-431817564189/tests/tzhty_t8yeqcda9x5dzkay6j5bczzwgxffz_5hwy/node_modules_stream.zip to s3://artilleryio-test-data-431817564189/tests/tzhty_t8yeqcda9x5dzkay6j5bczzwgxffz_5hwy/node_modules.zip	-
January 25, 2024 at 17:06 (UTC+3:00)	Modules pre-packaged	-
January 25, 2024 at 17:06 (UTC+3:00)	Installing required npm dependencies	-
January 25, 2024 at 17:06 (UTC+3:00)	Wrote to /artillery/test_data/package.json:	-
January 25, 2024 at 17:06 (UTC+3:00)	{	-
January 25, 2024 at 17:06 (UTC+3:00)	"name": "test_data",	-
January 25, 2024 at 17:06 (UTC+3:00)	"version": "1.0.0",	-
January 25, 2024 at 17:06 (UTC+3:00)	"description": "",	-
January 25, 2024 at 17:06 (UTC+3:00)	"main": "flows.js",	-
January 25, 2024 at 17:06 (UTC+3:00)	"scripts": {	-
January 25, 2024 at 17:06 (UTC+3:00)	"test": "echo \"Error: no test specified\" && exit 1"	-
January 25, 2024 at 17:06 (UTC+3:00)	},	-
January 25, 2024 at 17:06 (UTC+3:00)	"keywords": [],	-
January 25, 2024 at 17:06 (UTC+3:00)	"author": "",	-
January 25, 2024 at 17:06 (UTC+3:00)	"license": "ISC"	-
January 25, 2024 at 17:06 (UTC+3:00)	}	-
January 25, 2024 at 17:06 (UTC+3:00)	******** [1ca155c9bdaa4e40ad95f0b6b530cb62] Installing dependencies	-
January 25, 2024 at 17:06 (UTC+3:00)	******** [1ca155c9bdaa4e40ad95f0b6b530cb62] Test run ID = tzhty_t8yeqcda9x5dzkay6j5bczzwgxffz_5hwy	-
January 25, 2024 at 17:06 (UTC+3:00)	******** [1ca155c9bdaa4e40ad95f0b6b530cb62] Syncing test data	-
January 25, 2024 at 17:06 (UTC+3:00)	******** [1ca155c9bdaa4e40ad95f0b6b530cb62] Worker starting up, ID = 1ca155c9bdaa4e40ad95f0b6b530cb62, version = f7534a2844b58ee2a851081a3c498e671ad94f17, leader = true	-
```

**Reply by mrister** [reply]
Did you try to follow these instructions (note the new worker image) https://github.com/artilleryio/artillery/discussions/2426#discussioncomment-8170747

**Reply by En3Tho** [reply]
Just tried it (just the docker image part as I'm running from main anyway) and it worked. I guess I was kinda silly of me not to try that before I posted here. Thank you.

**Comment by buenofc** [comment]
it seems the **Error: Timed out waiting for workers to sync** is back! I'm running artillery v2.0.19 with AWS Fargate with 1 user (1 worker). I tried increase the memory (--memory 16) to 16GB from 8GB, but no joy. Any help with will be much appreciated!!!

`Artillery Cloud reporting is configured for this test run
Run URL: https://app.artillery.io/load-tests/tdfqw_rp6n679rgmf5dc6ndjnpnq3b9zpwn_96z8
Checking AWS connectivity... [11:05:27]
Checking cluster... [11:05:28]
Tags: name:shine_bc_smoke.yml [11:05:29]
Test run ID: tdfqw_rp6n679rgmf5dc6ndjnpnq3b9zpwn_96z8 [11:05:29]
Preparing launch platform... [11:05:29]
Environment:
  Account:     841162689491
  Region:      ap-southeast-2
  Count:       1
  Cluster:     artilleryio-cluster
  Launch type: ecs:fargate (On-demand)

Preparing test bundle... [11:05:30]
| =============

Test bundle prepared... [11:05:32]
Test bundle contents:
┌───────────────────────────────────────────────────────────┬─────────┬───────┐
│ Name                                                      │ Type    │ Notes │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ performance\shine_bc_smoke.yml                            │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ performance\processor.ts                                  │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ support\pages\BasePage.ts                                 │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\login\LoginPage.ts                           │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\home\HomePage.ts                             │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\customer\CustomerCard.ts                     │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\helpers\helpers.ts                           │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\sales\SalesOrdersPage.ts                     │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\purchase\PurchaseOrdersPage.ts               │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\components\TopBar.ts                         │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\runs\RunsCard.ts                             │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\statement\StatementRunCard.ts                │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\report\IncomeStatementDialog.ts              │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\report\PaymentsOnHoldDialog.ts               │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\report\AgedAccPayBackDatingDialog.ts         │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\report\AgedAccountsReceivableDialog.ts       │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\report\AgedAccRecBackDatingDialog.ts         │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\report\CustomerListDialog.ts                 │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\report\credit-memo\PurchaseCreditNotePage.ts │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\report\BankAccountLedgerEntriesPage.ts       │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ models\pages\index.ts                                     │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\purchase.to.pay.ts                               │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\run.regular.sales.orders.ts                      │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\statement.run.ts                                 │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\post.released.sales.orders.ts                    │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\reports\income.statement.ts                      │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\reports\payments.on.hold.ts                      │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\reports\aged.acc.pay.backdating.ts               │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\reports\aged.acc.rec.backdating.ts               │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\reports\customer.list.ts                         │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\reports\aged.accounts.receivable.ts              │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\sales.to.invoice.ts                              │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\reports\purchase.credit.memo.ts                  │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ commands\reports\apd.eft.listing.ts                       │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ data\users.csv                                            │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ data\sales.orders.csv                                     │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ data\purchase.orders.csv                                  │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ data\statement.run.csv                                    │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ data\run.regular.sales.orders.csv                         │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ package.json                                              │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┤       │
│ package-lock.json                                         │ file    │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ @eslint/compat                                            │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ @eslint/js                                                │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ @playwright/test                                          │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ @types/eslint__js                                         │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ @types/node                                               │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ @types/uuid                                               │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ artillery                                                 │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ artillery-engine-playwright                               │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ dotenv                                                    │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ eslint                                                    │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ eslint-config-prettier                                    │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ eslint-plugin-prettier                                    │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ foreach                                                   │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ playwright                                                │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ playwright-ts-artillery-performancetesting                │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ prettier                                                  │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ typescript                                                │ package │       │
├───────────────────────────────────────────────────────────┼─────────┼───────┤
│ typescript-eslint                                         │ package │       │
└───────────────────────────────────────────────────────────┴─────────┴───────┘

Launching workers... [11:05:35]
Waiting for Fargate... [11:05:36]
Waiting for workers to start: pending: 1 [11:06:06]
Waiting for workers to start: pending: 1 [11:06:16]
Waiting for workers to start: pending: 1 [11:06:26]
Waiting for workers to start: pending: 1 [11:06:37]
Waiting for workers to come online... [11:06:37]
Error: Timed out waiting for workers to sync
Error: Timed out waiting for workers to sync
    at C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\artillery\lib\platform\aws-ecs\legacy\run-cluster.js:1862:29
    at finished (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\artillery\lib\platform\aws-ecs\legacy\util.js:109:16)
    at C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\async\dist\async.js:972:16
    at next (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\async\dist\async.js:3025:18)
    at Response.<anonymous> (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\artillery\lib\platform\aws-ecs\legacy\util.js:96:20)
    at Request.<anonymous> (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\request.js:367:18)
    at Request.callListeners (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\sequential_executor.js:106:20)
    at Request.emit (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\sequential_executor.js:78:10)
    at Request.emit (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\request.js:686:14)
    at Request.transition (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\request.js:22:10)
Error: Timed out waiting for workers to sync
    at C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\artillery\lib\platform\aws-ecs\legacy\run-cluster.js:1862:29
    at finished (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\artillery\lib\platform\aws-ecs\legacy\util.js:109:16)
    at C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\async\dist\async.js:972:16
    at next (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\async\dist\async.js:3025:18)
    at Response.<anonymous> (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\artillery\lib\platform\aws-ecs\legacy\util.js:96:20)
    at Request.<anonymous> (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\request.js:367:18)
    at Request.callListeners (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\sequential_executor.js:106:20)
    at Request.emit (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\sequential_executor.js:78:10)
    at Request.emit (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\request.js:686:14)
    at Request.transition (C:\Users\ebueno\OneDrive - Planit Test Management Solutions Pty Ltd\git\ampol-project-shine-bc-artillery-playwright\node_modules\aws-sdk\lib\request.js:22:10)
Cleaning up... [11:16:32]
/ 

Run URL: https://app.artillery.io/load-tests/tdfqw_rp6n679rgmf5dc6ndjnpnq3b9zpwn_96z8`
